.gitlab-ci.yml

```diff
@@ -1,6 +1,25 @@
+stages:
+  - test
 
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-commits.yml'
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-check-license.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-python3-format.yml'
+
+test:static:
+  extends: .test
+  stage: test
+  image: python:3.6-buster
+  before_script:
+    - pip install isort
+    - pip install mypy
+    - pip install pylint
+    - pip install .
+    - pip install -r requirements.txt
+  script:
+    - isort --check-only --line-length 88 --wrap-length 88 examples
```

Comment:
Love it!
:medal_military: 
We've as a company have become a lot better at using these sorts of tools lately I feel :)

Reply:
Well... you introduced these! I am only following your path :running_man: